Add exchange integration guide to SDK docs (#1014)
iotaledger/iota-wiki: shimmer/docs/wallet.rs/docs/_admonitions/_warning_unlock_conditions.md

If you are synchronizing outputs with other unlock conditions, you should check the unlock conditions carefully before crediting users any balance.

> You can find an example illustrating how to check if an output has only the address unlock condition, where the address belongs to the account in the [Check Unlock Conditions how-to guide](https://github.com/iotaledger/iota-sdk/blob/develop/bindings/nodejs/examples/wallet/17-check-unlock-conditions.ts).

Reviewer: ```suggestion
You can find an example illustrating how to check whether an output has only the address unlock condition and whether such address belongs to the account in the [Check Unlock Conditions how-to guide](https://github.com/iotaledger/iota-sdk/blob/develop/bindings/nodejs/examples/wallet/17-check-unlock-conditions.ts).
```

> You can find an example illustrating how to check whether an output has only the address unlock condition and whether this address belongs to the account in the [Check Unlock Conditions how-to guide](https://github.com/iotaledger/iota-sdk/blob/develop/bindings/nodejs/examples/wallet/17-check-unlock-conditions.ts).

Author: This isn't needed in the shimmer docs. 

```suggestion
You can find an example illustrating how to check whether an output has only the address unlock condition and whether this address belongs to the account in the [Check Unlock Conditions how-to guide](../how_tos/outputs_and_transactions/06_check_unlock_conditions.mdx).
```

Reviewer 2: @Dr-Electron but in the shimmer wiki there isn't the sdk, so how does this all match?

Author: The SDK isn't released yet. So thy do we care? On Shimmer you should still use wallet.rs and iota.rs

Reviewer 3: No they should use the SDK, it'll be release by then

Author: I got it. But this was just a wrong replacement in the Shimmer part of the wiki instead of next where the SDK docs currently are located 👍
